webpack.mix.js

```diff
@@ -12,4 +12,7 @@ const mix = require('laravel-mix');
  */
 
 mix.js('resources/js/app.js', 'public/js')
-    .sass('resources/sass/app.scss', 'public/css');
+    .sass('resources/sass/app.scss', 'public/css')
+    .postCss("resources/css/app.css", "public/css", [
+        require("tailwindcss"),
+   ]); 
```

Comment:
eof

Reply:
dạ cái này CI nó check failed do eof ấy a

Reply:
FILE: /build/awesome-academy/Naitei-202307-E-Learning/webpack.mix.js
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
18 | ERROR | [x] Whitespace found at end of line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY